app/settings/data-export/data-export.controller.js

```diff
@@ -54,6 +56,16 @@ function (
         }
     });
 
+    // Check if hxl-api-key is added
+    UserSettingsEndpoint.getFresh({id: $rootScope.currentUser.userId}).$promise.then((settings) => {
+        $scope.hxlApiKey = false;
+        _.each(settings, (setting) => {
```

Comment:
you should iterate over `settings.results ` to get the items in settings (sorry if someone covered this already in another PR )

Reply:
Yep, true, fixed now